samples/SessionSample/Startup.cs

```diff
@@ -17,7 +16,7 @@ public Startup(ILoggerFactory loggerFactory)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddCaching();
-            services.AddSessionServices();
+            services.AddSession();
```

Comment:
Hmm. I didn't move any services with this change. Its just rename. I will check on that.